testing-utils/package.json

```diff
@@ -1,5 +1,32 @@
 {
   "name": "@code-pushup/testing-utils",
   "version": "0.5.0",
+  "license": "MIT",
+  "homepage": "https://github.com/code-pushup/cli#readme",
+  "bugs": {
+    "url": "https://github.com/code-pushup/cli/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/code-pushup/cli.git",
+    "directory": "packages/testing-utils"
+  },
```

Comment:
`package.json` is needed for buildable and publishable Nx projects.

Which other projects are never intended to be published? Maybe we could add `"private": true` to the `package.json` files of these projects to make it explicit.

Reply:
Only the projects in `packages` are intended to be published. I think `testing-utils` is the only other project with a `package.json`, other non-publishable projects are in `e2e` and `examples`.

The `testing-utils` project is buildable because otherwise the `@nx/enforce-module-boundaries` lint rule fails with `Buildable libraries cannot import or export from non-buildable libraries`. But I don't believe it needs to be publishable, I tried deleting its `package.json` now and it doesn't seem to cause any problems.

Reply:
>  I tried deleting its `package.json now` and it doesn't seem to cause any problems.

It's not recommend to delete `package.json` files from buildable projects.

Reply:
Changes to the `testing-utils` project have been omitted from this PR. `esbuild.config.js` controls which projects to add the `package.json` metadata to as part of the build process. Relevant metadata is copied from the root `package.json` file.